.travis.yml

```diff
@@ -29,3 +29,18 @@ matrix:
       - env: CPP_DOC_CHECK
         install: sudo apt-get install -y doxygen
         script: cd docs/cpp/source && ./check-doxygen.sh
+      - env: CLANG_TIDY
```

Comment:
NB: Since we're moving to CircleCI, I'm going to ask you to redo this in our CircleCI config later, when it goes live. CC @yf225

Reply:
That's ok